lib/Sema/CSDiag.cpp

```diff
@@ -3611,6 +3654,24 @@ static bool diagnoseArgumentLabelError(Expr *expr,
   
   // Emit Fix-Its to correct the names.
   auto &diag = *diagOpt;
+  
+  // If we're attempting to fix only the first parameter,
+  // offer up the same fix as a scalar tuple promotion.
+  if (numMissing == 1 && declIsInModule &&
+      tuple->getElementName(0) != newNames[0]) {
```

Comment:
`tuple->getElementName()` and `newNames[0]` are `Identifier`s -- what does that do with `!`?